### ISSUE TITLE
exclude files from folder

### ISSUE THREAD

**Issue by dimitre** [chalet-org/chalet#202]
Is it possible to exclude certain files from folder in a build?
I'm tring this
```json
			"files": {
				"include": [ "../libs/openFrameworks/**" ],

				"exclude": [
					"../libs/openFrameworks/video/ofGstUtils.*",
					"../libs/openFrameworks/video/ofAVFoundationVideoPlayer.*",
					"../libs/openFrameworks/video/ofGstVideoPlayer.*"
				]	
			 }
```
but it seems files aren't excluded from build. or maybe include is being executed after exclude.
thank you

**Comment by dimitre** [comment]
Yes I now see why a relative path can be a problem. It will be great to be supported too, since we cannot create projects in a folder not empty
```
ERROR: Path '/Volumes/tool' is not empty. Please choose a different path, or clean this one first.
```

**Comment by rewrking** [comment]
No, that willl never be a thing. `init` is meant for creating fresh projects from scratch. I would just make an empty folder, create a project, and then use what you need in the other path. It just creates a hello world and a minimal `chalet.json`.

**Comment by rewrking** [comment]
exact file excludes and relative paths are fixed in [v0.5.6](https://github.com/chalet-org/chalet/releases/tag/v0.5.6)

Let me know if you have any other issues!

**Comment by dimitre** [comment]
Tested and fixed! Thank you
